docs/querying/sql-functions.md

```diff
@@ -1414,11 +1562,32 @@ Returns the number of the row within the window starting from 1.
 
 ## RPAD
 
-`RPAD(<CHARACTER>, <INTEGER>, [<CHARACTER>])`
+Returns a string of size `length` from `expr`, right-padded with `chars`. The default value for `chars` is a space character. Truncates `expr` to `length` if `length` is shorter than the length of `expr`.
 
-**Function type:** [Scalar, string](sql-scalar.md#string-functions)
+* **Syntax:** `RPAD(expr, length[, chars])`
+* **Function type:** Scalar, string
+
+<details><summary>Example</summary>
+
+The following example inserts `+` characters as padding to the right of the `OriginState` column from the `flight-carriers` datasource, making the return expression a length of `5` characters.
```

Comment:
why does the length of the return string matter?

Reply:
I included it because the definition includes: "Returns a string of size `length`", so I though it important to point it out in the definition. I do not think it is absolutely necessary, but I still think its useful to distinguish that its different than just insert x characters to the front/back of the string

Reply:
Ignoring